en/peripherals/uavcan_escs.md

```diff
@@ -0,0 +1,98 @@
+# UAVCAN ESCs (Motor Controllers)
+
+PX4 supports the [UAVCAN](https://uavcan.org/) bus for connecting peripherals, including ESCs, GPS modules, various types of sensors, etc.
+
+UAVCAN ESCs have a number of advantages over [PWM ESCs and Servos](../peripherals/pwm_escs_and_servo.md):
+- UAVCAN has been specifically designed to deliver robust and reliable connectivity over relatively large distances.
+  It enables safe use of ESCs on bigger vehicles and communication redundancy.
+- The bus is bi-directional, enabling health monitoring and diagnostics.
+- Wiring is less complicated as you can have a single bus for connecting all your ESCs and other UAVCAN peripherals.
+- Setup is easier as you configure ESC numbering by manually spinning each motor.
+
+PX4 can be used with UAVCAN ESC that run [Sapog](https://github.com/PX4/sapog#px4-sapog) firmware; an advanced open source sensorless PMSM/BLDC motor controller firmware designed for use in propulsion systems of electric unmanned vehicles.
```

Comment:
This is misleading. PX4 can be used with any UAVCAN ESC, not only Sapog-based (e.g., VESC, Zubax Myxa, etc).